cicecore/cicedynB/dynamics/ice_dyn_vp.F90

```diff
@@ -908,14 +915,15 @@ subroutine anderson_solver (icellt  , icellu, &
                !$OMP PARALLEL DO PRIVATE(iblk)
                do iblk = 1, nblocks
                   ! first compute diagonal contributions due to rheology term
-                  call formDiag_step1 (nx_block             , ny_block      , &
-                                       icellu       (iblk)  ,                 &
-                                       indxui     (:,iblk)  , indxuj(:,iblk), &
-                                       dxt      (:,:,iblk)  , dyt (:,:,iblk), &
-                                       dxhy     (:,:,iblk)  , dyhx(:,:,iblk), &
-                                       cxp      (:,:,iblk)  , cyp (:,:,iblk), &
-                                       cxm      (:,:,iblk)  , cym (:,:,iblk), &
-                                       zetaD    (:,:,iblk,:), diag_rheo(:,:,:))
+                  call formDiag_step1 (nx_block           , ny_block      ,    &
+                                       icellu (iblk)      ,                    &
+                                       indxui (:,iblk)    , indxuj(:,iblk),    &
+                                       dxt    (:,:,iblk)  , dyt (:,:,iblk),    &
+                                       dxhy   (:,:,iblk)  , dyhx(:,:,iblk),    &
+                                       cxp    (:,:,iblk)  , cyp (:,:,iblk),    &
+                                       cxm    (:,:,iblk)  , cym (:,:,iblk),    &
+                                       zetax2 (:,:,iblk,:), etax2(:,:,iblk,:), &
+                                       diag_rheo(:,:,:))
```

Comment:
Can we avoid changing the whitespace in these lines ? it also adds noise to the diff so it's harder to spot what is really changing...